src/System.Threading.ThreadPool/ref/System.Threading.ThreadPool.cs

```diff
@@ -23,7 +23,11 @@ public static partial class ThreadPool
         public static void GetMinThreads(out int workerThreads, out int completionPortThreads) { throw null; }
         public static bool QueueUserWorkItem(System.Threading.WaitCallback callBack) { throw null; }
         public static bool QueueUserWorkItem(System.Threading.WaitCallback callBack, object state) { throw null; }
+#if uap
```

Comment:
Just add it to the API compat baseline instead of adding ifdefs?

Reply:
Yep that worked great.